controllers/configmap_controller.go

```diff
@@ -186,7 +187,21 @@ func (r *ConfigMapReconciler) Reconcile(ctx context.Context,
 	// At this point configMap will be set properly
 	switch req.NamespacedName.Name {
 	case servicescm.Name:
-		return ctrl.Result{}, r.reconcileServices(ctx, configMap)
+    // get the machineConfig
+    machineConfig := &mcfgv1.MachineConfig{}
+    if err := r.client.Get(ctx, kubeTypes.NamespacedName{
+        Namespace: req.Namespace}, machineConfig); err != nil {
+        return ctrl.Result{}, err
+    }
+    node := &core.Node{}
+    if err := r.client.Get(ctx, kubeTypes.NamespacedName{
+      Namespace: "", 
+      Name: req.Name},
```

Comment:
which node would you get by doing this? I thought the req.Name would be the name of a configmap given the usage of `handler.EnqueueRequestsFromMapFunc(r.mapToServicesConfigMap)` in your added Watches